### DISCUSSION TITLE
open directory of current pdf

### DISCUSSION THREAD

**Discussion by maribox** [ahrm/sioyek#1096] [Q&A] [answered]
Hey, how to open the directory of the currently opened document in the default file manager?
Is that possible?

**Comment by ahrm** [comment, marked as answer]
There is currently no command to do that.

**Reply by maribox** [reply]
can i open an issue or is this not wanted?

**Reply by ahrm** [reply]
I have already implemented this in my private sioyek repository, it will be available when the next version is released.
